packages/getting-started/src/browser/getting-started-contribution.ts

```diff
@@ -16,8 +16,10 @@
 
 import { injectable, inject } from '@theia/core/shared/inversify';
 import { CommandRegistry, MenuModelRegistry } from '@theia/core/lib/common';
-import { CommonMenus, AbstractViewContribution, FrontendApplicationContribution, FrontendApplication } from '@theia/core/lib/browser';
+import { CommonMenus, AbstractViewContribution, FrontendApplicationContribution, FrontendApplication, PreferenceService } from '@theia/core/lib/browser';
+import { EditorManager } from '../../../editor/lib/browser';
```

Comment:
I'm surprised this works :)

```suggestion
import { EditorManager } from '@theia/editor/lib/browser';
```

You'll also need to add the dependency to the package's `package.json`. 

---
Alternatively you can prevent coupling between packages by using `ApplicationShell`.

```ts
const editors = this.shell.widgets.filter((widget): widget is NavigatableWidget => NavigatableWidget.is(widget));
```